easybuild/easyconfigs/p/PyCUDA/PyCUDA-2017.1.1-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'PyCUDA'
+version = '2017.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mathema.tician.de/software/pycuda'
+description = """PyCUDA lets you access Nvidia’s CUDA parallel computation API from Python."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['6b5a9384e38c603ee429f8a6bee424532db7b3505027ce22f7e18ad19564b563']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('CUDA', '9.1.85', '', True),
+    ('Boost', '1.67.0', versionsuffix),
```

Comment:
@SimonPinches For easyconfigs using a `2018a` toolchain, we should stick to `Boost` 1.66.0` for now, to ensure that this module is compatible with others that depend on `Boost` (cfr. failing test)...